src/client/components/dashboard/Mining.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { Card, CardDeck, Container, Row, Col } from "react-bootstrap";
+
+
+class Mining extends Component {
```

Comment:
nit: can you please name this component more intuitive, e.g. MiningSection, MiningPage ?